packages/react-reconciler/src/ReactFiberReconciler.old.js

```diff
@@ -265,6 +267,10 @@ export function updateContainer(
   const suspenseConfig = requestCurrentSuspenseConfig();
   const lane = requestUpdateLane(current, suspenseConfig);
 
+  if (enableSchedulingProfiler) {
+    markRenderScheduled(current, lane);
```

Comment:
Sorry I missed this earlier, but doing a final review before merging and noticed the `current` param is still being passed (but not used). Can we remove it?